### ISSUE TITLE
Basic Definitions for Solid-OIDC Providers

### ISSUE THREAD

**Issue by danielbakas** [solid/solid-oidc#227]
Hi everyone!

* **TL;DR:** Suggestion to add a _SHOULD_ or _MUST_ in the spec for declaring basic information like the label or icon of a Solid-OIDC.

After a very interesting and productive conversation with @elf-pavlik about retrieving basic information like a Solid-OIDC provider's label and icon based on a WebID, I've found that there is no standard for our providers to declare this basic information.

I would like to suggest the addition to a few but standard and useful predicates to a Solid-OIDC WebID document, such as: label, description, icon, and those that you think would be valuable.

These labels and icons could be used in our applications' user interfaces to improve user experience in many scenarios such as:

* Choosing the Solid-OIDC provider for Login/Registration
* Rendering information about Solid-OIDC providers programatically, reducing the need for manual information retrieval
* & more

We'd love to know what you think 🤓

In the meantime thanks and cheers 🥳

**Comment by elf-pavlik** [comment]
Currently one can only rely on information from https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata

We are discussing adopting OIDC Federation  https://openid.net/specs/openid-federation-1_0.html#name-openid-provider

it seems to have `logo_uri`.

I wonder what others think about using issuer IRI as WebID and getting a human-readable label, logo, and other useful descriptions from there.
